src/vs/platform/dnd/browser/dnd.ts

```diff
@@ -23,6 +23,9 @@ import { ByteSize, IFileService } from '../../files/common/files.js';
 import { IInstantiationService, ServicesAccessor } from '../../instantiation/common/instantiation.js';
 import { extractSelection } from '../../opener/common/opener.js';
 import { Registry } from '../../registry/common/platform.js';
+// eslint-disable-next-line local/code-import-patterns
+import { IHostService } from '../../../workbench/services/host/browser/host.js';
```

Comment:
this does the trick, but we can't import `IHostService` into `platform/dnd/browser`

any suggestions?